assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -103,6 +103,10 @@ module.exports = elementorModules.ViewModule.extend( {
 	},
 
 	showModal: function( options ) {
+		if ( options.url ) {
+			options.url = options.url.replaceAll( 'javascript:', '' );
```

Comment:
It can be in other formats, it needs a better escape.

Reply:
- Only for URLs that been created by Elementor (starts with `http`)
- It might break custom link made by users (like relative links)

Reply:
Done.